### DISCUSSION TITLE
Deployment error: azurerm_key_vault_access_policy

### DISCUSSION THREAD

**Discussion by kenleung-PW** [microsoft/PubSec-Info-Assistant#784] [Q&A]
Hi,

I attempted to deploy the version 1.1.1 of the Info Assistant, but encountered the following error after applying the Terraform plan:

![error](https://github.com/microsoft/PubSec-Info-Assistant/assets/106700772/55b92246-b268-4e3e-92fc-8c05e83b305c)

After I got this error, I deleted the resource group and the resources created and confirmed that they are removed. I also looked under "Key Vaults" to confirm the key vault created from the previous run no longer exists. Then I deploy again, but the same error is shown.

Please advise and let me know how to proceed with the deployment. Thank you.

**Comment by dayland** [comment]
KeyVaults and several other resources such as AI Services, AI Search, etc. go into a "soft deleted" state but are not in fact deleted. They appear in this state so they can be recovered or fully purged. The best option is to update your workspace value in `local.env` to target a net new resource group. The "soft deleted" service will eventually be purged after several days (each service has its own soft delete retention period).
